src/sshfs_mount/sshfs_mount.cpp

```diff
@@ -830,7 +830,7 @@ mp::SshfsMount::SshfsMount(std::function<SSHSession()> make_session, const std::
                            const std::string& target, const std::unordered_map<int, int>& gid_map,
                            const std::unordered_map<int, int>& uid_map, std::ostream& cout)
     : ssh_session{make_session()},
-      sshfs_process{create_sshfs_process(ssh_session, source, target)},
+      sshfs_process{create_sshfs_process(ssh_session, mp::utils::escape_char(source, '"'), target)},
```

Comment:
We should also escape the target since it's possible someone would do:

``bin/multipass mount /tmp/dquotes\"\ spaces foo``

which leads to the same bash failure.

Reply:
Oh right! done.